homeassistant/components/airzone_cloud/climate.py

```diff
@@ -278,9 +304,58 @@ def __init__(
         ]
         if HVACMode.OFF not in self._attr_hvac_modes:
             self._attr_hvac_modes += [HVACMode.OFF]
+        if (
+            self.get_airzone_value(AZD_SPEED) is not None
+            and self.get_airzone_value(AZD_SPEEDS) is not None
+        ):
+            self._set_fan_speeds()
 
         self._async_update_attrs()
 
+    def _set_fan_speeds(self) -> None:
+        """Initialize Aidoo fan speeds."""
+        azd_speeds: dict[int, int] = self.get_airzone_value(AZD_SPEEDS)
+        max_speed = max(azd_speeds)
+
+        speeds: dict[int, str]
+        if _speeds := FAN_SPEED_MAPS.get(max_speed):
+            speeds = _speeds
+        else:
+            speeds = {}
```

Comment:
I don't know if its possible, but can we give the names some more context? We now have `speeds`, `_speeds`, `self._speeds` and its kinda confusing

Reply:
Done in cdbe317185844534075504d610aec0a92beb640e